pkgs/top-level/all-packages.nix

```diff
@@ -35066,6 +35066,8 @@ with pkgs;
 
   rootlesskit = callPackage ../tools/virtualization/rootlesskit { };
 
+  rosa = callPackage ../applications/networking/cluster/rosa { };
```

Comment:
Move it to the pkgs/by-name hierarchy

Reply:
done. since this is fairly new I didn't know this existed. I also found the [docs](https://github.com/NixOS/nixpkgs/commits/master/pkgs/by-name/README.md) about it. Thanks for your patience and guidance